frappe_graphql/utils/resolver/document_resolver.py

```diff
@@ -67,8 +67,10 @@ def _get_value(fieldname, ignore_translation=False):
 
         # ignore_doc_resolver_translation might be helpful for overriding document_resolver
         # which might be a simple wrapper around this function (document_resolver)
+        field_name_exits = meta.get_field(info.field_name)
```

Comment:
```suggestion
        df = meta.get_field(info.field_name)
```

Reply:
@fahimalizain cannot do that df is declared and assigned in the outer scope 🥶

Reply:
@e-lobo why not re-use the same variable ? it is accessible right ?

Reply:
@fahimalizain im not sure i can do that. check the usages of `_get_value(fieldname)` the `fieldname` is unpredictable. please clarify ?

Reply:
@e-lobo yes, nice catch 👍🏼 
I would say you could rename it to `_df` or something at least.  `field_name_exits ` can only ever be a `boolean` 😄

Reply:
@fahimalizain updated ✅